src/services/models/Schema.ts

```diff
@@ -139,6 +139,8 @@ export class SchemaModel {
       this.items = new SchemaModel(parser, schema.items, this._$ref + '/items', this.options);
       this.displayType = this.items.displayType;
       this.typePrefix = this.items.typePrefix + 'Array of ';
+      this.format = this.items.format;
```

Comment:
Could you remove this line? Schema model is used not only for rendering. It may be used for other purposes so copying `format` field from `items` is not quite valid.

The correct approach would be to introduce `displayFormat` field (similar to `displayType`) which is used only for rendering purposes.

Reply:
done. Do you think the same "rule" should be applied for the `title` field?

Reply:
Nice! Thanks!

> Do you think the same "rule" should be applied for the title field?

I don't think so.